### PR TITLE
corrected acording to the comments

### PR DESCRIPTION
**Pull request by OleksandrNovitskyi**
version 1.1

### CODE REVIEW

**Review by auraz — CHANGES_REQUESTED** [review]
Do we need a media folder and its contents?

**Comment by OleksandrNovitskyi** [comment]
> Do we need a media folder and its contents?

I already delete it. I check photos by link to LinkedIn

**Comment by auraz** [comment]
> I already delete it. I check photos by link to LinkedIn

Sorry, not seen in the mobile app that files were deleted.

File: sort_names.py [hunk 1]
```diff
@@ -1,33 +1,56 @@
 import csv
+from fileinput import filename
```
Comment (auraz) [inline sort_names.py:2]: As a learning point: remember about order of imports and PEP8

**Review by auraz — APPROVED** [review]

**Pull request merged**